content/millennium/r4/care-provision/care-team.md

```diff
@@ -0,0 +1,140 @@
+---
+title: CareTeam | R4 API
+---
+
+# CareTeam
+
+* TOC
+{:toc}
+
+## Overview
+
+The R4 CareTeam resource can be used to represent care teams.
+
+The CareTeam resource represents a patient’s care team members. Care team members or participants include practitioners (physicians, nurses, technicians, etc.), family members, friends, guardians, and the patient. The care team can be specific to an encounter or to the patient across all encounters (longitudinal).
+
+The following fields are returned if valued:
+
+* [Care Team id](http://hl7.org/fhir/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status (Active)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.status){:target="_blank"}
+* [Category (Longitudinal or Encounter)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.category){:target="_blank"}
+* [Name (Encounter-Level Only)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.name){:target="_blank"}
+* [Subject](http://hl7.org/fhir/careteam-definitions.html#CareTeam.subject){:target="_blank"}
+* [Encounter (Encounter-Level Only)]( http://hl7.org/fhir/careteam-definitions.html#CareTeam.encounter){:target="_blank"}
+* [Participant](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant){:target="_blank"}
+  * [Role](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.role){:target="_blank"}
+  * [Member](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.member){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:care_team, :r4) %>
+
+
+## Search
+
+Search for CareTeams that meet supplied query parameters:
+
+    GET /CareTeam?:parameters
+
+_Implementation Notes_
+
+* The [CareTeam.activity] element is not supported and will not be returned. This means that the [CareTeam.activity.detail.status] and [CareTeam.activity.detail.prohibited] modifier elements will not be returned.
+
+* The description of the CareTeam is included in the [CareTeam.text.div] element as ASCII text.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Parameters
+
+ Name         | Required? | Type          | Description
+--------------|-----------|---------------|--------------
+ `_id`        | See notes | [`token`]     | The logical resource id associated with the resource.
+ `patient`    | See notes | [`reference`] | Who care team is for. Example: `patient=12345`
+ `encounter`  | See notes | [`reference`] | Who care team is for. Example: `encounter=98765`
+ `category`   | N         | [`token`]     | The scope of care team being searched for. Examples: `category=assess-team`
+ `status`     | N         | [`token`]     | Indicates the status of the care team
+
+
+Notes:
+
+- The `_id` parameter
+  - This is required if patient or encounter are not provided.
+
+- The `patient` parameter
+  - This is required when `_id` is not provided.
```

Comment:
the _id section says patient or encounter is needed, these sections say each is required

Reply:
Addressed. https://github.com/cerner/fhir.cerner.com/pull/470/commits/1c1aad456a43d4ba5ae6e7fc8620a05d19202987